src/brand/product.hb

```diff
@@ -14,6 +14,20 @@
         <p>For guidance on picking variations or positioning logos, take a look at <a href="/brand/logo">the page for the Redgate logo</a> - it's mostly the same advice.</p>
         <p>Get in touch with <a href="mailto:brand@red-gate.com">brand@red-gate.com</a> if you have any questions.</p>
 
+        <div class="grid__col grid__col--span-6-of-12 padded-v">
+            <a href="src/assets/brand/product-logos/flyway.zip" class="link-image">
```

Comment:
Typo on this line. Need to remove the `src` at the start of the hyperlink's `href` value.